zipline/utils/cache.py

```diff
@@ -20,7 +20,7 @@ class Expired(Exception):
     """
 
 
-class CachedObject(namedtuple("_CachedObject", "value expires")):
```

Comment:
@llllllllll and I had some concerns about the behavior of garbage collection on objects with empty slots.  See the discussion in https://bugs.python.org/issue24379.

The specific issue, I think, is that objects with empty `__slots__` don't participate in garbage collection, which means they can leak if they participate in a cycle.  I don't think that's happening here, but there's also not much benefit in making this a subclass of namedtuple.